collections/_projects/freifunk-on-satellites.md

```diff
@@ -0,0 +1,70 @@
+---
```

Comment:
we need the metadata `size` here.. please set
`size: half-time` or `size: full-time`